core-plugin/src/com/google/gct/idea/debugger/actions/CloudDebugHelpAction.java

```diff
@@ -0,0 +1,32 @@
+package com.google.gct.idea.debugger.actions;
+
+import com.intellij.CommonBundle;
+import com.intellij.ide.BrowserUtil;
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.actionSystem.Presentation;
+import com.intellij.openapi.util.IconLoader;
+
+import javax.swing.Icon;
+
+/**
+ * Opens the Cloud Debugger help page.
+ */
+public class CloudDebugHelpAction extends AnAction {
+  private static final Icon icon = IconLoader.getIcon("/actions/help.png");
+  private String url;
+
+  public CloudDebugHelpAction(String url) {
+    this.url = url;
+  }
+
+  public void actionPerformed(AnActionEvent event) {
```

Comment:
Again, I'm OK with just a unit test for the current case (URL not set) that will fail as soon as we set that parameter (likely next week).